charts/common/values.yaml

```diff
@@ -144,15 +144,17 @@ service:
   # The name of the service is optional, If not set, a name is generated using Release.Name
   # name: ""
 
-ingress:
+privateIngress:
   enabled: false
   hosts:
     - host: chart-example.local
       # The name is optional, If not set, a name is generated using host
       # name: ""
-      annotations: {}
-        # kubernetes.io/ingress.class: (nginx-internal or nginx-public)
-        # kubernetes.io/tls-acme: "true"
+      annotations: 
+        cert-manager.io/cluster-issuer: letsencrypt
+        kubernetes.io/ingress.class: nginx-internal
```

Comment:
If we're going to call this privateIngress, can we force this annotation by default?